internal/commands/login/inputs_test.go

```diff
@@ -237,4 +253,19 @@ func TestLoginInputs(t *testing.T) {
 		assert.Equal(t, "username", i.Username)
 		assert.Equal(t, "password", i.Password)
 	})
+
+	t.Run("should error if browser flag is enabled and credentials are input", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+		_, ui := mock.NewUI()
```

Comment:
probably don't even need the mock UI here (and can pass in `nil` to the `Resolve` func)